### DISCUSSION TITLE
acesso remoto

### DISCUSSION THREAD

**Discussion by luis007007** [community/community#132251] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

olá, pessoal, eu estou centro controlado remotamente, alguem conseguiria me ajudar?, nao estou conseguindo trabalhar,

**Comment by rrsayao** [comment, marked as answer]
Consegue especificar melhor o seu problema?

**Comment by LiteBrite82** [comment]
Hi @luis007007 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
